client/locales/de.json

```diff
@@ -0,0 +1,402 @@
+{
+    "common": {
+        "ok": "OK",
+        "submit": "Übernehmen",
+        "cancel": "Abbrechen",
+        "overwrite": "Überschreiben",
+        "continue": "Weiter",
+        "dismiss": "Abbrechen",
+        "prompt": "Hinweis",
+        "warning": "Warnung",
+        "attention": "Achtung!",
+        "confirm": "Bestätigen",
+        "close": "Schließen",
+        "save": "Speichern",
+        "cancelled": "Abgebrochen",
+        "enabled": "Eingeschaltet",
+        "commandOK": "Befehl erfolgreich übertragen!",
+        "commandFailed": "Es ist ein Fehler aufgetreten:",
+        "connectionError": "Verbindungsfehler",
+        "m2": "m²",
+        "hours": "Stunde",
+        "hours_plural": "Stunden",
+        "months": ["Jan","Feb","Mär","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dez"],
+        "days": ["Montag","Dienstag","Mittwoch","Donnerstag","Freitag","Samstag","Sonntag"]
+    },
+    "navi": {
+        "home": "Home",
+        "map": "Karte",
+        "zones": "Zonen",
+        "control": "Steuerung",
+        "settings": "Einstellungen",
+        "links": "Links",
+        "about": "Über"
+    },
+    "home": {
+        "labels": {
+            "area": "Bereich",
+            "time": "Zeit",
+            "battery": "Batterie",
+            "state": "Ladungszustand..."
+        },
+        "buttons": {
+            "start": "Start",
+            "pause": "Pause",
+            "stop": "Stop",
+            "home": "Home",
+            "spot": "Spot",
+            "find": "Finde",
+            "goto": "Gehe zu",
+            "zones": "Zonen",
+            "storeMap": "Karte speichern",
+            "loadMap": "Karte laden",
+            "unknownPower": "Unbekannte Leistung",
+            "invalid": "Ungültiger Button"
+        },
+        "popups": {
+            "selectFanspeed": "Auswahl des Leistungsmodus:",
+            "selectGotoTarget": "Setzte einen Zielpunkt:",
+            "selectZones": "Auswahl der Zonen zur Reinigung:",
```

Comment:
"selectZones": "Bereiche für Reinigung wählen:"

Reply:
This is changing all zone and zones descriptions. I will keep Zone but will rephrase it accordingly.